utils/metadata_utils.py

```diff
@@ -0,0 +1,55 @@
+    skull_image = np.array(Image.open(BytesIO(base64.b64decode(skull_image_base64))))
+
+    confidence = 0.666
+    skull_image = skull_image[::2, ::2]
+    frame = frame[::2, ::2]
+
+    result = cv2.matchTemplate(frame, skull_image, cv2.TM_CCOEFF_NORMED)
+    match_indices = np.arange(result.size)[(result > confidence).flatten()]
+    matches = np.unravel_index(match_indices[:10000], result.shape)
+
+    return len(matches[0]) != 0
+
+
+def generate_metadata(frame: list):
+    """
+    Generates metadata for a given frame.
+
+    :param frame: The frame to process.
+    :return: tuple of (fov, azimuth, elevation).
+    """
+
+    fov = random.choice([8.954, 74.654, 24.65])
+    azimuth = random.choice([-22.58, 36.54, 1.45])
+    elevation = random.choice([11.862, 69.42, 78.4])
+
+    return fov, azimuth, elevation
+
+
+def get_metadata_arguments(image: list):
```

Comment:
Please dont add type hints or change it from list to np.ndarray,
I thought at first that its a list of images.